javascript/modules/client/src/internal/client-core.ts

```diff
@@ -103,13 +117,29 @@ export abstract class ClientCore implements IClientCore {
     return this._daoFactoryAddress;
   }
 
+  get ipfs() {
+    return this._ipfs;
+  }
+
+  get ipfsIdx() {
```

Comment:
Getters should have friendly names. How about `ipfsIndex`?
Indeed... do we really need to export this? Consumers care about getting a client, not about an index. See above